pkg/polaris-reporting/ctl_polarisreporting.go

```diff
@@ -65,6 +65,25 @@ type FlagTree struct {
 	SMTPTlsTrustedHosts       string
 }
 
+// DefaultFlagTree ...
+// [Dev Note]: These should match the Helm Chart's Values.yaml
+var DefaultFlagTree = FlagTree{
+	// version
+	Version: "2020.04",
+	// domain specific flags
+	IngressClass: "nginx",
+	// license related flags
+	// storage related flags
+	ReportStorageSize: "5Gi",
+	EventstoreSize:    "50Gi",
+	// smtp related flags
+	// postgres specific flags
+	PostgresInternal: "false",
+	PostgresPort:     5432,
+	PostgresSSLMode:  "require",
+	PostgresSize:     "50Gi",
+}
```

Comment:
Enable Persistent storage to true?

Reply:
There isn't a flag for enablePersistentStorage